src/libs/API/parameters/UpdateSageIntacctNonreimbursableExpensesCreditCardChargeExportDefaultVendorParams.ts

```diff
@@ -0,0 +1,6 @@
+type UpdateSageIntacctNonreimbursableExpensesCreditCardChargeExportDefaultVendorParams = {
+    policyID: string;
+    vendorID: string;
+};
+
+export default UpdateSageIntacctNonreimbursableExpensesCreditCardChargeExportDefaultVendorParams;
```

Comment:
Similar comment as with https://github.com/Expensify/App/pull/44166#discussion_r1669106233, let's use the same pattern added in that PR